# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wgpu/.github/workflows/docs.yml
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/gfx/wgpu/.github/workflows/docs.yml b/gfx/wgpu/.github/workflows/docs.yml
deleted file mode 100644
--- a/gfx/wgpu/.github/workflows/docs.yml
+++ /dev/null
@@ -1,45 +0,0 @@
-name: Documentation
-
-on:
-  push:
-    branches:
-      - master
-
-jobs:
-  build:
-    runs-on: ubuntu-latest
-
-    steps:
-      - name: Checkout the code
-        uses: actions/checkout@v2
-        with:
-          persist-credentials: false
-
-      - name: Install latest nightly
-        uses: actions-rs/toolchain@v1
-        with:
-          toolchain: nightly
-          override: true
-        continue-on-error: true
-
-      - name: Add EGL for OpenGL
-        run: |
-          sudo apt-get update -y -qq
-          sudo apt-get install -y -qq libegl1-mesa-dev
-
-      - name: Build the docs (nightly)
-        run: |
-          cargo +nightly doc --lib --all-features
-
-      - name: Build the docs (stable)
-        run: cargo +stable doc --lib --all-features
-        if: ${{ failure() }}
-
-      - name: Deploy the docs
-        uses: JamesIves/github-pages-deploy-action@releases/v3
-        with:
-          ACCESS_TOKEN: ${{ secrets.WEB_DEPLOY }}
-          FOLDER: target/doc
-          REPOSITORY_NAME: gfx-rs/wgpu-rs.github.io
-          BRANCH: master
-          TARGET_FOLDER: doc 